src/components/tooltip/tooltip.js

```diff
@@ -165,7 +164,7 @@ class Tooltip extends Component {
         <span
           style={{maxWidth: props.maxWidth}}
           ref={el => {
```

Comment:
why use this patten?
Also, see other comment about ref - better use the `createRef` preact utility for this.